src/models/enums.ts

```diff
@@ -0,0 +1,25 @@
+export enum Grade{
```

Comment:
I suggest adding a default `Unknown` value in case we encounter an unknown value.